google/cloud/bigquery/_helpers.py

```diff
@@ -172,3 +172,55 @@ def _validate(self, value):
         """
         if value not in self.ALLOWED:
             raise ValueError('Pass one of: %s' ', '.join(self.ALLOWED))
+
+
+class UDFResource(object):
```

Comment:
I agree that it probably should have been in a different PR. I don't think it's worth going back for though.
If you don't squash merge then we could always cherry-pick that commit if needed. Just my $0.02